task/04-arrays-tasks.js

```diff
@@ -415,7 +438,11 @@ function toStringList(arr) {
  *      { country: 'Russia',  city: 'Saint Petersburg' }
  */
 function sortCitiesArray(arr) {
-   throw new Error('Not implemented');
+    const sorted = arr.sort((a, b) => {
+        return a.country === b.country ? a.city.localeCompare(b.city) : a.country.localeCompare(b.country);
```

Comment:
I would prefer `if` statement in this case because of the readability, but ternary is ok